lib/utils/env_vars.js

```diff
@@ -13,11 +13,19 @@ const env_vars = {
   PLAYGROUND_EMAIL: process.env.PLAYGROUND_EMAIL,
   PLAYGROUND_PASSWORD: process.env.PLAYGROUND_PASSWORD,
   BASE_URL: process.env.BASE_URL,
-  BRANCH_NAME: process.env.BRANCH_PATH && process.env.BRANCH_PATH.split('/')[1] || 'test',
   REPO_URL: process.env.REPO_URL,
   DEBUG: process.env.DEBUG,
 };
 
+// Path could be 'refs/branch_name' or 'refs/head/branch_name' and maybe other things
+// Assume branch name is last part of branch path
+if ( BRANCH_PATH ) {
```

Comment:
I think this should be `process.env.BRANCH_PATH`, since the variable by itself doesn't exist yet.

Reply:
Gorramit. You're right. Good eye!

-